style.css

```diff
@@ -0,0 +1,558 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+.wrapper {
+	max-width: 930px;
+	min-width: 320px;
+	font-family: 'GothamPro', serif;
+	margin: 0 auto;
+	box-sizing: border-box;
+}
+
+/*header*/
+
+.header {
+	width: 100%;
+	height: 96px;
+	display: grid;
+	flex-direction: row;
+	grid-template-columns: 0.7fr 2.7fr 1fr 1.5fr 0.6fr 0.6fr 0.3fr;
+	margin-bottom: 34px;
+}
+
+.menu-bar {
+	display: none;
+}
+
+.header__logo {
+	display: flex;
+	align-items: center;
+}
+
+.header__logo-image {
+	width: 55px;
+	height: 55px;
+}
+
+.header__logo-image_adaptive {
+	display: none;
+	width: 156px;
+	height: 34px;
+}
+
+.header__menu {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	/*padding-top: 48px;
+	padding-bottom: 24px;*/	
+}
+
+.link {
+	text-decoration: none;
+	color: #000000;
+	text-transform: uppercase;
+	font-size: 12px;
+	font-weight: bold;
+	line-height: 11px;
+}
+
+.header__search {
+	display: flex;
+	position: relative;
+	justify-content: center;
+	align-items: center;
+	width: 160px;
+}
+
+.search-input {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+	border: none;
+	border-bottom: 1px solid #000000;
+}
+
+.search-icon {
+	display: flex;
+	position: absolute;
+	left: 148px;
+	align-items: center;
+	justify-content: center;
+	width: 14px;
+	height: 14px;
+}
+
+.header__user-block {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	text-transform: uppercase;
+	font-weight: bold;
+}
+
+.header__cart {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.header__cart_counter {
+	height: 14px;
+	width: 14px;
+	border-radius: 50%;
+	background-color: #000000;
+	color: #ffffff;
+	margin-right: -20px;
+	margin-left: 5px;
+}
+
+.header__language-block {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+	margin: 0;
+	padding: 0;
+}
+
+.header__language-image {
+	width: 20px;
+	height: 20px;
+}
+
+/*top-content*/
+
+.top-content {
+	background-image: url(img/header-img.png);
+	display: flex;
+	justify-content: flex-start;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)